apps/flow.md

```diff
@@ -0,0 +1,46 @@
+# Flow
+How to be Good at Flow
+
+## Terms
+
+### Workspaces
+Workspaces contain all of the info for large projects where clients and contractors are allowed to see our process in action. These are the top level of the Flow hierarchy.
+
+### Projects
+Projects contain all of the info for a subset of a project in a workspace. For Good, this is how we split our internal projects from each other. For larger projects, this will typically be an Active Sprint project and a Backlog project.
+
+### Sections
+Sections split out projects into logic sections for things like To-Do, In Progress, etc.
+
+### Tasks
+Tasks are the smallest unit of the workflow and are moved in between sections and sometimes projects.
+
+### Tags
+Tags should be used to categorise tasks in a focused manner. Try not to exceed 3 tags on one task. Here are the common tags:
+
+* pm - project management
+* fe - front end
+* be - backend
+* fs - full stack
+* de - design
+* ux - user experience
+* ui - user interface
+* bd - business development
+* feature
+* bug
+* debt
+
+## Using Flow
+
+### Managing Tasks
+Tasks should represent a single thing that needs to be completed and can be accomplished within the span of a day. If a task spans multiple days, it needs to be broken down. Make sure you don't break down tasks too much. Twenty tasks to accomplish something that only takes an hour is obviously excessive. Tasks should be tagged and dated appropriately. If you know someone cares about the progress of a task, subscribe them to it. For lengthier tasks that don't need to be broken down, use subtasks so any subscribers have an idea of the progress that's been made.
+
+### Agile Workflows
+A typical agile workspace will have two projects: Active Sprint and Backlog.
+
+Active Sprint only concerns tasks related to the current sprint time period, which is typically 1 or 2 weeks. This project will have sections for To-Do, In Progress, Staging, and Release Candidate. Staging is for tasks that have been deployed to the staging server. Release Candidate covers tasks that have been approved and are ready to deploy to production. Only include the necessary sections based on what the project calls for.
+
+Backlog contains all of the non active tasks for a project. Sections for this project are usually Features, Bugs, Requests, and Tech Debt. Only include the necessary sections based on what the project calls for.
+
+### Overview Tab
```

Comment:
We should add something about prioritizing tasks that have due dates for today as well as looking ahead and tasks with due dates _this week_ so nothing sneaks up on you. After tasks with a due date are taken care of, you should just work on a project each day on a per-priority basis.